.helmignore

```diff
@@ -35,3 +35,7 @@ requirements.txt
 tests/
 venv/
 whitelist.txt
+
+# Helm dependency build generated files
+charts/*/charts
+chart.lock
```

Comment:
I think this might be `Chart.lock`? I don't have such a file, but this doc supports my theory https://helm.sh/docs/helm/helm_dependency_build/
```suggestion
Chart.lock
```